_api-reference/script-apis/exec-stored-script.md

```diff
@@ -125,4 +125,198 @@ The `GET books/_search` request returns the following fields:
 | _index | String | Index that contains the document. |
 | _id | String | Document ID. |
 | _score | Float | Document's relevance score. |
-| fields | Object | Fields and their value returned from the script. |
\ No newline at end of file
+| fields | Object | Fields and their value returned from the script. |
+
+## Execute Painless stored script with parameters
+
+To pass different parameters to the script each time when running a query, define `params` in `script_fields`.
+
+### Example
+
+The following request runs the stored script that was created in [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book, multiplies the summed-up value by the `multiplier` param, and displays the result in the output.
+
+* The script's target is the `books` index.
+
+* The `"match_all": {}` property value is an empty object indicating processing each document in the index.
```

Comment:
Is this the meaning? "... indicating that it processes each document in the index."